Trainee/Reviews/Reviews.jsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { useState } from "react";
+import { FaStar } from "react-icons/fa";
+import axios from "../../../api/axios";
+import ViewReviews from "./ViewReviews/ViewReviews";
+import { Scrollbars } from "react-custom-scrollbars";
+import "./Reviews.css";
+
+const reviewsUrl = "/review";
+
+export default function Review({ training, traineeName }) {
+  const [currentValue, setCurrentValue] = useState(0);
+  const [hoverValue, setHoverValue] = useState(undefined);
+  const [trainingType, setTrainingType] = useState("");
+  const [review, setReview] = useState("");
+  const date = new Date().toLocaleString("en-us", {
+    day: "numeric",
+    month: "short",
+    year: "numeric",
+  });
+  const starsNo = Array(5).fill(0);
+
+  const handleClick = (value) => {
+    setCurrentValue(value);
+  };
+  const handleMouseOver = (value) => {
+    setHoverValue(value);
+  };
+  const handleMouseLeave = () => {
+    setHoverValue(undefined);
+  };
+  const handleReviews = async (e) => {
+    e.preventDefault();
+    try {
+      const trainerName = training.fullname;
+
+      const response = await axios.post(
+        reviewsUrl,
+        JSON.stringify({
+          traineeName,
+          trainerName,
+          currentValue,
+          trainingType,
+          review,
+          date,
+        }),
+        {
+          headers: { "Content-Type": "application/json" },
+          withCredentials: true,
+        }
+      );
+      alert("Submitted!");
+    } catch (err) {
+      alert("Error!");
+    }
+  };
+
+  return (
+    <div>
+      <div className="view-reviews">
+        <ViewReviews trainerName={training.fullname} />
+      </div>
+      <div className="write-review">
+        <p>Write A Review for {training.fullname}</p>
+
+        {starsNo.map((_, index) => {
```

Comment:
should be ReviewStars.jsx component

Reply:
I have changed it to aRating component from material/ui